rollup.config.js

```diff
@@ -7,7 +7,10 @@ module.exports = {
 	exports: "none",
 	plugins: [
 		babel( {
-			presets: [ "es2015-rollup" ]
+			presets: [
+				[ "es2015", { modules: false } ]
```

Comment:
This is basically telling Babel to not transpile ES modules, as they will be handled by rollup.